core/trino-main/src/main/java/io/trino/operator/TableScanWorkProcessorOperator.java

```diff
@@ -217,7 +217,10 @@ DataSize getPhysicalInputDataSize()
 
         long getPhysicalInputPositions()
         {
-            return processedPositions;
```

Comment:
there is also `io.trino.operator.TableScanOperator` to be changed

Reply:
updated